flake.nix

```diff
@@ -45,7 +41,37 @@
       url = "github:michaelpj/sphinxcontrib-haddock";
       flake = false;
     };
+    tullia = {
+      url = "github:input-output-hk/tullia";
+      # inputs.nixpkgs.follows = "nixpkgs";
+    };
+    utils.url = "github:numtide/flake-utils";
   };
 
-  outputs = _: { };
+  outputs = { self, utils, tullia, ... } @ inputs:
+    let
+      supportedSystems = [ "x86_64-linux" "x86_64-darwin" ];
+    in
+      utils.lib.eachSystem supportedSystems (system:
+        {
+          # TODO
+          # do it similar to cardano-node?
+          # or use flattenTree and filter by .system/.meta.platforms?
+          #   might be an option because there doesn't seem to be IFD in here (in contrast to
+          #   cardano-node)
+          #   but might be bad idea because it will break if they ever add IFD
+
+          # Does not follow flake output schema, only created to pass to tullia
+          hydraJobs = import ./ci.nix {
```

Comment:
It doesn't need to be a valid set of hydra jobs. `flakeOutputTasks` just expects some arbitrarily-nested attrset that contains derivations. In fact, it cannot be: The flake output schema is `.#hydraJobs.<attr-maybe-nested>.<system> = drv`. But in order to evaluate which jobs exist we may need IFD that builds on another system. So instead I opted to put the system first and set `supportedSystems` to only the respective system, so that we can safely select `.#hydraJobs.<system>` and recurse into that.

Reply:
That all sounds sensible. I also like putting the system first!